doc/admin/troubleshoot.rst

```diff
@@ -254,6 +254,10 @@ This issue can be fixed in two ways:
 -   Manually: :ref:`reseed <replication-master-master-reseed-replica>` one master from another by removing write-ahead logs and snapshots.
 -   Programmatically: set up a :ref:`conflict resolution trigger <replication-problem_solving>`.
 
+..  NOTE::
+
+    To avoid brain split between two masters, one of the instances may be put in the :ref:`isolated mode <_configuration_reference_isolated_mode>`.
```

Comment:
It gives a false impression that the isolated mode somehow helps to avoid split-brain, but it means that we should detect is somehow faster than tarantool itself. It seems strange.

Here we can fight with consequences of split-brain to some extent. At least, isolate the broken instance from client requests using the config option.

Let's discuss how to better express it.